maltd-frontend/src/components/composite/UserSearch/UserSearch.stories.js

```diff
@@ -5,11 +5,15 @@ import "bootstrap/dist/css/bootstrap.css";
 import UserSearch from "./UserSearch";
 
 const userSearch = {
-  state: { isLoading: false }
+  state: { isLoading: false, userExists: true }
```

Comment:
`userExists` should have 3 possible states which are
1) null -> meaning the user has not entered anything yet and hasnt made a request with an idir -> this is the default state
2) true -> meaning the user has entered and searched based on an idir that is valid
3) false -> meaning the user has entered and searched based on an idir that is invalid